forest/src/daemon.rs

```diff
@@ -354,6 +354,12 @@ async fn sync_from_snapshot(config: &Config, state_manager: &Arc<StateManager<Ro
         } else {
             Some(0)
         };
+
+        if config.client.halt_after_import {
+            info!("Forest finish shutdown");
+            process::exit(0);
```

Comment:
@elmattic Do you think it'll cause problems with RocksDB if we call `exit(0)` here? Do we need to properly shutdown the database?

Reply:
Sure we should avoid this and this is possible to do (from my experience, the main source of resource leaks are situated after the import).

But more problematic with this PR is that the code exits before `import_chain` in case of user `--halt-after-import`, resulting in a no-op. We need to do it after the import.

Plus we need also to run this code:
```
   match chain_store.heaviest_tipset().await { ...
```
This is needed to check that the user is not importing to old snapshots.

If you do just after:
```
    // Halt
    if config.client.halt_after_import {
        info!("Forest finished import");
        return
    }
```
this should terminate forest properly. Also make sure you check rocksdb LOG file to verify that it closed properly the db (you need to enable rocksdb debug log level).

Reply:
I think we could also promote this log `debug!("Imported snapshot in: {}s", ...` to an `info!` one.

This is useful information to know for troubleshooting, and it won't really bloat our logs.